libwild/src/input_data.rs

```diff
@@ -229,6 +205,37 @@ impl Input {
     }
 }
 
+pub(crate) fn mmap_file(path: &PathBuf, prepopulate_maps: bool) -> Result<Mmap> {
```

Comment:
Might be slightly nicer to take a `&Path` here rather than a `&PathBuf`. Requiring a PathBuf means that the caller needs to heap-allocate.

Reply:
Done